server/base/src/main/java/org/apache/accumulo/server/client/BulkImporter.java

```diff
@@ -236,12 +236,7 @@ public void run() {
         }
 
         // remove map files that have no more key extents to assign
-        Iterator<Entry<Path,List<KeyExtent>>> afIter = assignmentFailures.entrySet().iterator();
-        while (afIter.hasNext()) {
-          Entry<Path,List<KeyExtent>> entry = afIter.next();
-          if (entry.getValue().size() == 0)
-            afIter.remove();
-        }
+        assignmentFailures.entrySet().removeIf(entry -> entry.getValue().size() == 0);
```

Comment:
may be able to use `assignmentFailures.values()` here

Reply:
Is there an isEmpty() instead? Perhaps: `.removeIf(entry.getValue()::isEmpty());`?

Reply:
my update uses isEmpty()